server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -717,6 +717,19 @@ public static IndexMergePolicy fromString(String text) {
         Property.IndexScope
     );
 
+    /**
+     * Index setting describing the maximum number of uncommitted translog files at a time.
+     * If breached the shard will be Refreshed, this is to control large number of tranlog files
+     * downloads during recovery.
+     */
+    public static final Setting<Integer> INDEX_MAX_UNCOMMITTED_TRANSLOG_FILES = Setting.intSetting(
+        "index.max_uncommitted_translog_files",
```

Comment:
lets move this to cluster setting. I assume we need this setting only for remote store indexes. Lets change the name to factor in the same.

Reply:
Done